src/pages/DailyIncrease/index.js

```diff
@@ -0,0 +1,125 @@
+import * as c3 from "c3";
```

Comment:
How about naming this DailyIncreaseChart? This makes it a noun rather than a verb. 

It's what the method was named and it makes it consistent with naming the files for what they are creating (like OutbreakMap and TrendChart)